Intro to data (#3)
LaunchCodeEducation/data-analysis-curriculum: content/intro-to-data/reading/project-life-cycle/_index.md

Below are the key questions you want to answer during this stage:
- What are the relevant sources of data?
- How can you access that data?
- What are the guidelines / regulations around collecting the data

## Cleaning and Exploration

After data has been collected, it will most often need to be cleaned. Since collected data may often be improperly formatted, duplicated, have errors or typos, missing values, or simply incorrect, correcting these mistakes or “cleaning” the data is a very logical step. One of the most important reasons to clean data is so that you do not reach false or inaccurate conclusions when presenting it.

Exploring or investigating data is an important step in understanding it. Exploratory analysis may often be referenced as exploratory data analysis (EDA) which will be covered more in depth later in the course. Finding connections and similarities within your data often comes from exploration, which will help you with other methods that include cleaning, organization, and visualization.

Below are the key questions you want to answer during this stage:

> - What issues need to be addressed

Reviewer: ```suggestion
- What issues need to be addressed?
```

- What are the anolomies within the data?
- What trends or movements arrise?
- What presumptions or premise can be formed?

## Building a Model

> After the data has been organized you can begin to build out your model(s) to uncover insights related to your primary objectives. The goal of this step is to supply stakeholders with actionable insights that will fit into the directives given at the inception of the project.

Reviewer: ```suggestion
After the data has been organized, you can begin to build out your model(s) to uncover insights related to your primary objectives. The goal of this step is to supply stakeholders with actionable insights that will fit into the directives given at the inception of the project.
```

Below are the key questions you want to answer during this stage:

> - What model or technique best works for your scenario

Reviewer: ```suggestion
- What model or technique best works for your scenario?
```

- How complex does the model need to be?

## Model Validation

Is your model performing as expected? Are you gathering the data you want? These are questions you need to answer during this stage of the project. If you are not receiving the expected or desired outcomes from your chosen model or technique you may need to re-think the approach.

Below are the key questions you want to answer during this stage:
- Does the model need to be changed?
- Is the model accurate?
- Does it produce the desired outcomes?

## Visualization and Presentation

After collecting insights and deliverables you will need to present the data in a way that makes sense. Visualizing data makes it easier for everyone to understand but there are many strategies used to do so. Teams may choose to visualize their data with a simple chart, graphs, tables or create dashboards, bar charts, heat maps, pie charts, or histograms. There are numerous ways to visualize your data but knowing your target audience is important because it will help you convey your information or tell a story catered to them. Other key aspects of data visualization include:

> - `Accuracy`: Presenting data without distorting it is important! Be precise with trends or movements within the data so that nothing is misinterpreted.
> - `Clarity`: You want your visuals to provide a clear representation of your findings
> - `Tools`: Any tool used to create your visualizations needs to be well thought out, as others may be using these same tools when interacting with the data
> - `Accessibility`: Making sure your visualizations meet accessibility standards will accomodate a more diverse audience: things like text size, color specifications for individuals who might be color-blind or have low-vision.

Reviewer: ```suggestion
- Accuracy: Presenting data without distorting it is important! Be precise with trends or movements within the data so that nothing is misinterpreted.
- Clarity: You want your visuals to provide a clear representation of your findings
- Tools: Any tool used to create your visualizations needs to be well thought out, as others may be using these same tools when interacting with the data
- Accessibility: Making sure your visualizations meet accessibility standards will accommodate a more diverse audience: things like text size, and color specifications for individuals who might be color-blind or have low vision.
```

Below are the key questions you want to answer during this stage:
- What data is the most relevant / significant from the analysis
    - How do you most effectively communicate that
- What presentation method will be used
- What are the next steps?

### Data Analysis Lifecycle Visual

> 

Reviewer: I would add text to explain what this visual is so screen reader users will be able to understand what is happening.

Author: Added text to explain what is going on within the visual displayed at the bottom of the page.
